recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: a663dbe0feb04bd9949da315a3abc9ddf5ce84d4e07d16c5b96e0947ee56a4eb
 
 build:
-  number: 0
+  number: 1
```

Comment:
do we need build-number 1? was there #0 uploaded before?

Reply:
Initially, I was working on a build update to 0.24.1. When I saw that 0.24.2 had been released, I updated the version information and reset the build number to 0.